module/VuFind/tests/integration-tests/src/VuFindTest/Mink/HoldingsTest.php

```diff
@@ -82,17 +82,32 @@ function ($a) {
             $set
         );
 
-        return [...$msgSet, ...$groupSet, ...$allSet];
+        $totalSet = array_map(
+            function ($a) {
+                $a[] = true;
+                $a[] = true;
+                return $a;
+            },
+            [...$msgSet, ...$groupSet, ...$allSet]
+        );
+
+        $totalSet[] = array_merge($set[0], ['msg', false, true]);
```

Comment:
Maybe worth adding a comment here to explain why we're testing this exact combination of scenarios (if there's a specific reason).

Reply:
There is no specific reason. I just did not want to add tests for all the cases and selected three arbitrary ones.